extensions/sql-migration/src/dialog/validationResults/validateIrDialog.ts

```diff
@@ -14,6 +14,14 @@ import { getEncryptConnectionValue, getSourceConnectionProfile, getTrustServerCe
 
 const DialogName = 'ValidateIrDialog';
 
+enum HttpStatusCodes {
```

Comment:
this seems useful for other scenarios too. would it make sense to move out to a commons utils?

Reply:
I think it would make sense to reuse it if we have other areas that have the issue.  I would opt to move the code when adding other scenarios